src/components/UserCard.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    class="user-card"
+    @click.stop
+    @mouseover="$emit('mouseover', $event)"
+    @mouseleave="$emit('mouseleave', $event)"
+  >
+    <UserInfo :address="address" />
+  </div>
+</template>
+
+<script>
+import UserInfo from './UserInfo.vue';
+
+export default {
+  components: {
+    UserInfo,
+  },
+  props: {
+    address: { type: String, required: true },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.user-card {
+  top: 15px;
+  width: 75%;
+  border: 1px solid $card_border_color;
+  border-radius: 0.5rem;
+  position: absolute;
+  z-index: 1;
+  overflow: hidden;
+  cursor: default;
+  box-shadow: 4px 4px 4px 0 rgba(0, 0, 0, 0.3);
+  background-color: $thumbnail_background_color;
+}
+
+::v-deep .profile__stats {
+  display: grid;
+  grid-template-columns: 8rem auto;
+
+  .tip_stats {
+    grid-template-columns: auto;
+    order: 2;
+  }
+
+  .tips_stats_block {
+    border: 0;
+    border-bottom: 1px solid $light_color;
+    padding-top: 0.25rem;
+    padding-bottom: 0.25rem;
+  }
+
+  .stats {
+    border: 0;
+    border-right: 1px solid $light_color;
+    flex-direction: column;
+    justify-content: space-around;
+  }
+
+  .stat_block {
+    margin: 0;
+  }
+}
+
+::v-deep .profile__image {
+  .avatar {
+    width: 3.5rem;
+    height: 3.5rem;
+  }
+}
+
+@media screen and (max-width: 1024px) {
```

Comment:
Use [mobile mixin](https://github.com/aeternity/superhero-ui/blob/ded59a3317cbb4cb7f51fa78d8db462e879360a3/src/styles/_mixins.scss#L16-L20) instead, and put it inside `.user-card` like [here](https://github.com/aeternity/superhero-ui/blob/1df0e5c8aa9ea6c6df49b091af59ea1ac55585dc/src/views/TipsList.vue#L78-L80).